Dockerfile

```diff
@@ -1,7 +1,15 @@
 FROM --platform=$BUILDPLATFORM docker.io/golang:1.21-alpine3.18 as builder
 
+ARG TARGETPLATFORM
+ARG BUILDPLATFORM
+ARG TARGETOS
+ARG TARGETARCH
+
+ENV CGO_ENABLED=0
+ENV GO111MODULE=on
+
 # hadolint ignore=DL3018
-RUN apk update && apk add --no-cache \
+RUN uname -a && apk update && apk add --no-cache \
```

Comment:
From now onwards, we need this info to see what platform is the context of build stages